pyproject.toml

```diff
@@ -123,6 +123,7 @@ types-greenlet = "^3.1.0.20241221"
 types-pexpect = "^4.9.0.20241208"
 types-unidiff = "^0.7.0.20240505"
 uv = ">=0.6.2"
+licensecheck = { version = "^2025.1.0", markers = "python_version < '4.0'" }
```

Comment:
Based on the readings, python 4 will never come as per their creators. So what's the point of this check?